app/components/PackageView.js

```diff
@@ -134,6 +134,10 @@ export default class PackageView extends React.Component {
     let displayName = component ? component.content.name : packageInfo.name;
     let flags = packageInfo.flags || [];
 
+    if (flags.includes("Deprecated")) {
+      return;
```

Comment:
I don't think we should return from render. We should instead filter out this package when it comes back from the server.

Reply:
Done 👍🏽